contents/docs/feature-flags/local-evaluation.mdx

```diff
@@ -221,18 +147,43 @@ PostHog::getFeatureFlag(
     // Include any person properties, groups, or group properties required to evaluate the flag
     [
         'your_group_type' => 'your_group_id',
-        'another_group_type' => 'your_group_id'
+        'another_group_type' => 'another_group_id'
     ], // groups
     ['property_name' => 'value'], // person properties
     [
         'your_group_type' => ['group_property_name' => 'value'], 
-        'another_group_type' => ['group_property_name' => 'value']
+        'another_group_type' => ['group_property_name' => 'another value']
     ], // group properties
     false, // only_evaluate_locally, Optional. Defaults to false. Set to true if you don't want PostHog to make a server request if it can't evaluate locally
     true // sendFeatureFlagEvents
 );
 ```
 
+```csharp
```

Comment:
why not put this in the multilanguage component with the other examples?

Reply:
That’s a typo. Doh!

Reply:
Oh wait, it is in the Multilanguage component with the others. This is a new addition to  "Step 3: Evaluate your feature flag".